src/igvfd/audit/primary_cell.py

```diff
@@ -0,0 +1,20 @@
+from snovault.auditor import (
+    audit_checker,
+    AuditFailure,
+)
+from .formatter import (
+    audit_link,
+    path_to_text,
+)
+
+
+@audit_checker('PrimaryCell', frame='object')
```

Comment:
this is the same function repeated 3 times, we need to refactor (put it in one place) and call it in three different places.

Reply:
moving audit to biosample abstract level

Reply:
moving back to selective biosamples